### DISCUSSION TITLE
conda-forge package for babshka

### DISCUSSION THREAD

**Discussion by petrushy** [babashka/babashka#1492] [Ideas]
[ To keep development of this project going, consider sponsoring. If you are
already a sponsor, thank you! ]

**Is your feature request related to a problem? Please describe.**
Streamlined development environments that can be specified in an environment file.

**Describe the solution you'd like**
Create conda-forge package for babashka

**Describe alternatives you've considered**
There are alternatives that works, but in data science, science and python-related stuff conda is becoming a popular packaging for isolated environments. https://conda-forge.org/

**Additional context**
Preferably one should build the package in conda-feedstock CI rather than download binaries

**Comment by borkdude** [comment]
I won't maintain any package managers except brew and manual linux install scripts, but users are free to maintain alternative package managers themselves if they like (e.g. Pitch is maintaining a package for asdf, and so on).. As such, I'll convert this issue to a discussion, so people who want to do this can chime in.

**Reply by petrushy** [reply]
Fully understand. :)

**Comment by seowalex** [comment]
Hi, I'm interested in picking this up, and I believe conda-forge requires building from scratch. Where would be a good point of reference for this? Currently, I am looking at the GitHub workflow and Dockerfile.

**Reply by borkdude** [reply]
Just looking at what happens in CI should be your reference
